.env.production

```diff
@@ -1,11 +0,0 @@
-REACT_APP_VERSION=$npm_package_version
-REACT_APP_NAME=$npm_package_name
-REACT_APP_FIREBASE_KEY=AIzaSyBT7D0KjQgOu5C2cfJ1zTrk7G1GXHvHJSc

```

Comment:
This doesn't fix the problem. The keys are still in git history and so they're still public.

Reply:
Should make sure whatever keys that are being used are different